manifests/t/timedomain/acestudio/1.1.0/timedomain.acestudio.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: timedomain.acestudio
+PackageVersion: 1.1.0
+PackageLocale: en-US
+Publisher: timedomAin

```

```suggestion
Publisher: 北京时域科技有限公司
```
This must match Control Panel.